tf2_eigen/include/tf2_eigen/tf2_eigen.h

```diff
@@ -341,22 +341,6 @@ geometry_msgs::msg::Pose toMsg(const Eigen::Isometry3d& in) {
   return msg;
 }
 
-/** \brief Convert a Eigen::Vector3d type to a geometry_msgs::msg::Vector3.
```

Comment:
This shouldn't be removed. Needed by `collision_detection` submodule of moveit2.